examples/platforms/posix/flash.c

```diff
@@ -67,7 +65,12 @@ otError utilsFlashInit(void)
         mkdir("tmp", 0777);
     }
 
-    snprintf(fileName, sizeof(fileName), "tmp/%d_%d.flash", NODE_ID, (uint32_t)tv.tv_usec);
+    if (offset == NULL)
+    {
+        offset = "0";
+    }
+
+    snprintf(fileName, sizeof(fileName), "tmp/%d_%s.flash", NODE_ID, offset);
```

Comment:
I would prefer to put the offset first in the filename, that way the flash files are grouped by each test.